pkg/virt-controller/watch/util/util.go

```diff
@@ -118,3 +123,80 @@ func CreateDataVolumeManifest(clientset kubecli.KubevirtClient, dataVolumeTempla
 
 	return newDataVolume, nil
 }
+
+func SyncPausedConditionToPod(clientset kubecli.KubevirtClient, vmi *virtv1.VirtualMachineInstance, pod *corev1.Pod) error {
```

Comment:
I would prefer this will be placed in a better specific location over under a generic nameless `util`.

Can this fit in "kubevirt.io/kubevirt/pkg/controller" ?

Reply:
Right, also is only use by `pkg/virt-controller/watch/`   so it do not make sense to make it public, what do you think about move this to `pkg/virt-controller/watch/sync.go` and make it private again ?